WebRoot/templates/abook/Contacts.template

```diff
@@ -1109,26 +1109,26 @@
 <!------------------------------ ZmContactSearch ------------------------------>
 <!----------------------------------------------------------------------------->
 <template id='abook.Contacts#ZmContactSearch'>
-	<div id='${id}_preamble'></div>
 	<table role="presentation" width='100%'>
 		<tr>
 			<td>
-				<table role="presentation">
+				<table class='ZmContactSearchTable' role="presentation">
 					<tr>
-						<td width='20' valign='middle'><div id='${id}_searchIcon' class='ImgSearch'></div></td>
-						<td><input type='text' autocomplete='off' size='30' nowrap id='${id}_searchField'>&nbsp;</td>
-						<td id='${id}_searchButton'></td>
+						<td>
+							<input type='text' autocomplete='off' size='30' nowrap id='${id}_searchField'>
```

Comment:
Thanks to legacy code.
Fixed here.
https://github.com/Zimbra/zm-web-client/pull/117/commits/97cfa50df7bb0d1b69e5dbf78f80541dc101f0d3